plugin-api.d.ts

```diff
@@ -406,7 +406,6 @@ interface RunParametersEvent<ParametersType = ParameterValues | undefined> {
 }
 interface OpenDevResourcesEvent {
   command: 'open-dev-resource'
-  parameters?: undefined
```

Comment:
uh oh I think there might be something else that hasn't made its way into the plugins docs yet - do we want to keep this here or remove it?

Reply:
going to keep it in!